client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -232,7 +230,7 @@ class WpcomChecklistComponent extends PureComponent {
 	};
 
 	nextInlineHelp = () => {
-		const taskList = getTaskList( this.props );
+		const taskList = this.props.taskList;
```

Comment:
Note that this introduces a change that excludes tasks without a url, because that's what `getSiteTaskList` in the `connect` function now returns. I couldn't find any problems with this change, but I don't really understand when `nextInlineHelp` is used (or even if it's still used).